cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/TaxPage.java

```diff
@@ -0,0 +1,452 @@
+package com.skillstorm.PageObjects;
+
+
+
+
+import java.time.Duration;
+import java.util.List;
+
+
+
+
+import com.skillstorm.PageObjects.Interfaces.Component;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TaxPage extends Page {
+    private WebDriver driver;
+
+    public TaxPage(WebDriver driver) {
```

Comment:
Low impact: This constructor does not initialize the Navbar held within the Page class.
Suggestion: Initialize to a DashboardNavbar